akka-persistence/src/main/protobuf/MessageFormats.proto

```diff
@@ -45,3 +45,9 @@ message PersistentStateChangeEvent {
   required string stateIdentifier = 1;
   optional string timeout = 2;
 }
+
+message PersistentFSMSnapshot {
+  required string stateIdentifier = 1;
+  required PersistentPayload data = 2;
+  optional string timeout = 3;
```

Comment:
I see. What we can do with `PersistentStateChangeEvent` is to add another optional field and when deserializing we first look if the old string field is defined and then we try the new int field.

Perhaps those should be stored as nanoseconds in a `int64` so that we don't loose anything from the `FiniteDuration`?